recipe/meta.yaml

```diff
@@ -12,31 +12,32 @@ source:
 
 build:
   number: 8
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   skip: true  # [py<36]
 
 requirements:
   build:
-    - ninja
-    - cmake >=3.15
-    - make                                   # [unix]
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
     - {{ compiler('cxx') }}
+    - cmake >=3.15
+    - make  # [unix]
   host:
     - python
     - pybind11
```

Comment:
You might want to pin these to a specific version (greater than the minimum version specified in the project requirements) to enhance readability and future maintenance.

Reply:
FYI, the latest will do it (2.10.4)